images/rootfs.yml.in

```diff
@@ -1,11 +1,14 @@
 kernel:
   image: KERNEL_TAG
-  # the unified_cgroup_hierarchy forces cgroupsv1, which is required until pillar is ready to support v2
-  cmdline: "rootdelay=3 linuxkit.unified_cgroup_hierarchy=0"
+  # force cgroup v2
+  cmdline: "rootdelay=3 linuxkit.unified_cgroup_hierarchy=1 linuxkit.runc_console=1 linuxkit.runc_debug=1"
```

Comment:
Did you want to leave runc debugging and console enabled? Those are good for debugging, but it would be messy if left in permanently.

Also, it doesn't hurt to leave `linuxkit.unified_cgroup_hierarchy=1`, but you can remove it entirely, as this is the default.

Reply:
> I wanted to make it explicit

No problem there.